crates/ruff/src/rules/tryceratops/rules/reraise_no_cause.rs

```diff
@@ -55,10 +55,12 @@ pub(crate) fn reraise_no_cause(checker: &mut Checker, body: &[Stmt]) {
     };
 
     for (range, exc, cause) in raises {
-        if exc.map_or(false, |expr| expr.is_call_expr() && cause.is_none()) {
```

Comment:
Unrelated, I just find this very confusing (the `case.is_none()` has nothing to do with whether `exc` is `Some`).